packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -263,6 +263,32 @@ export class CDepositToVault extends ComposableBuffer<DepositToVault> {
   }
 }
 
+/**
+ * WithdrawFromVault DeFi Transaction
+ */
+export interface WithdrawFromVault {
+  vaultId: string // ------------------| 32 bytes, Vault Id
+  to: Script // ---------------------| n = VarUInt{1-9 bytes}, + n bytes, Address to receive withdrawn collateral
```

Comment:
alignment

Reply:
oops, they never align on my editor because some of my font